pkg/systemd/host.js

```diff
@@ -750,7 +750,12 @@ PageServer.prototype = {
         hardware_text.tooltip({ title: _("Click to see system hardware information"), placement: "bottom" });
         machine_info.dmi_info()
                 .done(function(fields) {
-                    hardware_text.text(fields.sys_vendor + " " + fields.product_name);
+                    let vendor = fields.sys_vendor || fields.board_vendor;
+                    let name = fields.product_name || fields.board_name;
```

Comment:
This means we might get "sys_vendor board_name".  Does that make sense?  Or is there some guarantee that if sys_vendor is present, so is product_name.